test/module/irohad/multi_sig_transactions/mst_processor_test.cpp

```diff
@@ -59,8 +60,11 @@ class MstProcessorTest : public testing::Test {
  protected:
   void SetUp() override {
     transport = std::make_shared<MockMstTransport>();
+    finalized_txs_ =
+        rxcpp::observable<>::empty<shared_model::interface::types::HashType>();
     storage =
         std::make_shared<MstStorageStateImpl>(std::make_shared<TestCompleter>(),
+                                              finalized_txs_,
```

Comment:
seems no need to store a copy of observable in test class, here and forth
```suggestion
                                              rxcpp::observable<>::empty<shared_model::interface::types::HashType>(),
```